.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: php
+sudo: false
```

Comment:
You should also include this line in readme:
`[![Travis CI](https://travis-ci.org/PayU/mysql-dump-anonymizer.svg)](https://travis-ci.org/PayU/mysql-dump-anonymizer)`

This way, the build status will be shown in the readme section. 
See this as an exmaple: https://raw.githubusercontent.com/PayU-EMEA/apple-pay/master/README.md

Reply:
done